app-containers/incus/incus-6.0.0-r1.ebuild

```
@@ -147,7 +152,12 @@ src_test() {
 
 src_install() {
 	export GOPATH="${S}/_dist"
-	local bindir="_dist/bin"
+
+	if [[ "${GOARCH}" == "arm64" ]]; then
```

I wonder if this shoudn't be more generic:

```suggestion
	if is_crosscompile; then
```

(from toolchain.eclass)